app/components/Dashboard/Cards/ResaleValue.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import GraphCard from "../../ui/GraphCard";
+import ResaleValueWrapper from "../../graphs/ResaleValueWrapper";
+import { Drawer } from "../../ui/Drawer";
+import { Household } from "@/app/models/Household";
+import TenureSelector from "../../ui/TenureSelector";
+
+interface DashboardProps {
+  data: Household;
+}
+
+export const ResaleValue: React.FC<DashboardProps> = ({ data }) => {
+  const [selectedTenure, setSelectedTenure] = useState<'landPurchase' | 'landRent'>('landPurchase');
```

Comment:
It's probably worth making a string union type for all tenures. This could then be `<Pick<Tenure, "landPurchase" | "landRent">>`